plugins/commander.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
+  shortDescription: interactive app for interacting with Kubernetes objects.
+  version: "v0.3.0"
+  description: |
+    kube-commander is an interactive menu (based on fzf) that allows you
+    to navigate and interact with object in your Kubernetes cluster. 
+    It is possible to quickly preview object configuration in YAML,
+    logs, events but also edite or delete object or enter into pods.
+  caveats: |
+    For optimal experience, be sure to have the following binaries 
+    installed on your machine:
+    fzf:  https://github.com/junegunn/fzf/releases
+    yq:  https://github.com/mikefarah/yq/releases
+  description: |
+    Usage:
```

Comment:
1. You're overwriting description above
2. This is not useful as people will go to your repo or -help flag to learn this anyway and this will go outdated of date.